lib/background.js

```diff
@@ -26,3 +26,13 @@ chrome.runtime.onMessage.addListener(({ urls }, sender) => {
     });
   });
 });
+
```

Comment:
After obtaining the new host permissions, we have to inject the content script in the matching pages. There are a few ways how to do that:

1. Use the [chrome.declarativeContent.RequestContentScript](https://developer.chrome.com/extensions/declarativeContent#type-RequestContentScript) action to insert a content script in the page.

2. Use the tabs API [chrome.tabs.onUpdated](https://developer.chrome.com/extensions/tabs#event-onUpdated) to detect that a page might have changed, and insert a content script in the page using [chrome.tabs.executeScript](https://developer.chrome.com/extensions/tabs#method-executeScript).

Unfortunately both are not working. 

The first option looks quite promising, but I found a bug which prevent any further https://bugs.chromium.org/p/chromium/issues/detail?id=646650

For the second option we have to ensure that our content script is designed in such a way that it can handle multiple insertions. The babel-polyfill throws an error if the content script is added more than once with `Only one instance of babel-polyfill is allowed`. I didn't found any workaround how to prevent that.